functional/FunctionalTest.java

```diff
@@ -1623,7 +1656,7 @@ public static void listIncompleteUploads_test1() throws Exception {
     try {
       String objectName = getRandomName();
       try (final InputStream is = new ContentInputStream(6 * MB)) {
-        client.putObject(bucketName, objectName, is, Long.valueOf( 9 * MB), null, null, nullContentType);
+        client.putObject(bucketName, objectName, is, new PutObjectOptions(9 * MB, -1));
```

Comment:
can we have few tests with `partSize` set to specific numbers  e.g. here with `new PutObjectOptions(9 * MB, 3)`

Reply:
`listIncompleteUploads_test1()` requires to have an incomplete (partial uploads). What do you mean by `PutObjectOptions(9 * MB, 3)`? Have you checked how those parameters mean to `PutObjectOptions()`?

Reply:
> Have you checked how those parameters mean to PutObjectOptions()?

I have checked. What I mean to say is I don't see a test case with `partSize` set to anything other than `-1`. We should have few tests where we pass `partSize` value to confirm if uploads work fine

Reply:
I am going to change few `PutObjectOptions()` with valid `partSize` on existing tests.

Reply:
thanks